src/Model/Item.php

```diff
@@ -1446,6 +1425,38 @@ public static function isValid(array $item)
 		return true;
 	}
 
+	/**
+	 * Check if the item array is too old
+	 *
+	 * @param array $item
+	 * @return boolean item is too old
+	 */
+	public static function tooOld(array $item)
```

Comment:
Please rename to `isTooOld` for consistency with `isValid`.

Reply:
Renamed